src/agimus/path_execution/play_path.py

```diff
@@ -42,73 +48,19 @@ def __str__(self):
         return repr(self.value)
 
 
-## Initialize the trajectory publisher.
-#
-# See agimus_hpp.trajectory_publisher.HppOutputQueue
-class InitializePath(smach.State):
-    hppTargetPubDict = {"read_subpath": [ReadSubPath, 1]}
-    serviceProxiesDict = {
-        "agimus": {"sot": {"clear_queues": [std_srvs.srv.Trigger]}},
-        "hpp": {"target": {"publish_first": [std_srvs.srv.Trigger]}},
-    }
-
-    def __init__(self):
-        super(InitializePath, self).__init__(
-            outcomes=_outcomes,
-            input_keys=[
-                "pathId",
-                "times",
-                "transitionIds",
-                "endStateIds",
-                "currentSection",
-            ],
-            output_keys=["transitionId", "endStateId", "currentSection", "duration"],
-        )
-
-        self.targetPub = ros_tools.createPublishers(
-            "/hpp/target", self.hppTargetPubDict
-        )
-        self.serviceProxies = ros_tools.createServiceProxies(
-            "", InitializePath.serviceProxiesDict
-        )
-        self.hppclient = HppClient(context="corbaserver")
-
-    def execute(self, userdata):
-        userdata.currentSection += 1
-        if userdata.currentSection + 1 >= len(userdata.times):
-            return _outcomes[0]
-        start = userdata.times[userdata.currentSection]
-        length = userdata.times[userdata.currentSection + 1] - start
-
-        transitionId = userdata.transitionIds[userdata.currentSection]
-        userdata.transitionId = transitionId
-        userdata.endStateId = userdata.endStateIds[userdata.currentSection]
-        userdata.duration = length
-
-        wait_if_step_by_step("Preparing to read subpath.", 3)
-
-        manip = self.hppclient._manip()
-        manip.graph.selectGraph(transitionId[1])
-        self.targetPub["read_subpath"].publish(
-            ReadSubPath(userdata.pathId, start, length)
-        )
-        rospy.loginfo("Start reading subpath.")
-        return _outcomes[2]
-
-
-## Execute a sub-path
+##_Execute a sub-path
 #
 # There are three main steps in execution of a sub-path
 # \li pre-action tasks
 # \li execution of the path
 # \li post-action tasks, typically closing the gripper
 #
 # Between each step, the execution is paused until a message on topic
-#  `/agimus/sot/event/done`.
-#  \todo fix scheduling, the current code waits a bit before considering
+# _`/agimus/sot/event/done`.
+# _\todo fix scheduling, the current code waits a bit before considering
```

Comment:
The unbreakable spaces were added by mistake because I use a french keyboard... Would you mind removing the `_` ?